src/lib/components/tooltip/tooltip.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { TooltipDirective } from './tooltip.directive';
+
+@NgModule({
+    imports: [],
+    declarations: [TooltipDirective],
+    exports: [TooltipDirective]
+})
+export class TooltipModule {}
```

Comment:
```suggestion
export class TerraTooltipModule {}
```
In order to be consistent with the other exported modules